### PR TITLE
Refactor/4/css styles and imports of components

### PR DESCRIPTION
**Pull request by ElenaCherpakova**
## One Line Description
- Set up jsconfig to shorten path lengths.
- Include an index.js file for every component.
- Eliminate unused styles and convert them to module.css.

## Test Steps
1. Once PR please run `npm i `to install dependencies.
2. Open the application.
3. Check if you don't have any warnings such as eslint and if the paths of some components can't be resolved or are undefined.

### CODE REVIEW

**Review by AnnaSolovykh — APPROVED** [review]
Great job!  
That is impressive! 👏🏻✨

File: src/components/Dashboard/Calendar/KFCalendar/DayEvent/DayEvent.jsx [hunk 1]
```diff
@@ -65,7 +43,7 @@ export const DayEvent = ({ event }) => {
           )}
         </div>
       </div>
-      <span className='circle' style={circleStyle}>
+      <span className={styles.circle} style={circleStyle}>
```
Comment (S-chahine) [inline src/components/Dashboard/Calendar/KFCalendar/DayEvent/DayEvent.jsx:46]: Update the <span> element with the following inline style to ensure centered content within .circle:
<span className={styles.circle} style={{
        ...circleStyle,
        display: 'flex',
        justifyContent: 'center',
        alignItems: 'center',
      }}>
        {kidsName && kidsName.charAt(0).toUpperCase()}
      </span>

File: src/components/Dashboard/Calendar/KFCalendar/WeekEvent/WeekEvent.jsx [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+      <div className={`${styles['content-wrapper']} content-wrapper}`}>
+        {truncatedTitle && (
+          <span className={styles.title}>{truncatedTitle}</span>
+        )}
+        <div className={styles.timing}>
+          {start && end && <span>{formatEventTime(start, end)}</span>}
+        </div>
+      </div>
+      <div></div>
+      <span className={`${styles.circle} circle`} style={circleStyle}>
```
Comment (S-chahine) [inline src/components/Dashboard/Calendar/KFCalendar/WeekEvent/WeekEvent.jsx:46]: Update the <span> element with the following inline style to ensure centered content within .circle:
<span className={styles.circle} style={{
        ...circleStyle,
        display: 'flex',
        justifyContent: 'center',
        alignItems: 'center',
      }}>
        {kidsName && kidsName.charAt(0).toUpperCase()}
      </span>

File: src/components/Dashboard/Calendar/KFCalendar/WeekEvent/WeekEvent.module.css [hunk 3]
```diff
@@ -0,0 +1,57 @@
+.title {
+  font-family: Roboto;
+  color: #081821;
+  font-size: 16px;
+  font-weight: 500;
+  line-height: 18.75px;
+  text-align: left;
+  margin-left: 0px;
+  margin-top: 10px;
```
Comment (S-chahine) [inline src/components/Dashboard/Calendar/KFCalendar/WeekEvent/WeekEvent.module.css:9]: adjust: margin-left: 15px;

File: src/components/Dashboard/Calendar/KFCalendar/DayEvent/DayEvent.module.css [hunk 4]
```diff
@@ -0,0 +1,74 @@
+.title {
+  font-family: Roboto;
+  color: #081821;
+  font-size: 16px;
+  font-weight: 500;
+  line-height: 18.75px;
+  text-align: left;
+  margin-left: 0px;
```
Comment (S-chahine) [inline src/components/Dashboard/Calendar/KFCalendar/DayEvent/DayEvent.module.css:8]: adjust
 margin-left: 5px;

File: src/components/Dashboard/Calendar/KFCalendar/DayEvent/DayEvent.module.css [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+  position: absolute;
+  top: 5px;
```
Comment (S-chahine) [inline src/components/Dashboard/Calendar/KFCalendar/DayEvent/DayEvent.module.css:10]: adjust 
top: -10px;

File: src/components/Dashboard/Calendar/KFCalendar/DayEvent/DayEvent.module.css [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+
+.desc {
+  font-family: Roboto;
+  color: #081821;
+  font-weight: 200;
+  font-size: 16px;
+  line-height: 40px;
+  display: flex;
+  flex-direction: column;
+  margin-left: 0;
```
Comment (S-chahine) [inline src/components/Dashboard/Calendar/KFCalendar/DayEvent/DayEvent.module.css:23]: adjust
margin-left: 5px;

File: src/components/Dashboard/Calendar/KFCalendar/DayEvent/DayEvent.module.css [hunk 7]
```diff
@@ -0,0 +1,74 @@
...
+  margin-bottom: -10px;
+  opacity: 0.75;
+}
+
+.dayEventContentWrapper {
+  display: flex;
+  flex-direction: column;
+  margin-left: 50px;
+  margin-top: 0px;
+}
```
Comment (S-chahine) [inline src/components/Dashboard/Calendar/KFCalendar/DayEvent/DayEvent.module.css:34]: add:
width: 100%;

File: src/components/Dashboard/Calendar/KFCalendar/DayEvent/DayEvent.module.css [hunk 8]
```diff
@@ -0,0 +1,74 @@
...
+
+.dayEventTiming {
+  font-size: 14px;
+  opacity: 0.4;
+  font-weight: 200;
+  line-height: 40px;
+  top: 10px;
```
Comment (S-chahine) [inline src/components/Dashboard/Calendar/KFCalendar/DayEvent/DayEvent.module.css:41]: adjust
top: 0px;

File: src/components/Dashboard/Calendar/KFCalendar/DayEvent/DayEvent.module.css [hunk 9]
```diff
@@ -0,0 +1,74 @@
...
+  margin-left: auto;
+  margin-right: 5px;
+}
```
Comment (S-chahine) [inline src/components/Dashboard/Calendar/KFCalendar/DayEvent/DayEvent.module.css:44]: add
position: relative;

**Review by S-chahine — CHANGES_REQUESTED** [review]
Great job Elena! We're almost there; just a bit of styling adjustment, and it'll be perfect.

File: src/components/Dashboard/Calendar/KFCalendar/WeekEvent/WeekEvent.jsx (hunk 2 above)
Comment (ElenaCherpakova) [reply src/components/Dashboard/Calendar/KFCalendar/WeekEvent/WeekEvent.jsx:46]: It was added into circle class in module.css for weekEvent

Comment (ElenaCherpakova) [reply src/components/Dashboard/Calendar/KFCalendar/WeekEvent/WeekEvent.jsx]: ✅ added to modules.css under the circle class

File: src/components/Dashboard/Calendar/KFCalendar/DayEvent/DayEvent.jsx (hunk 1 above)
Comment (ElenaCherpakova) [reply src/components/Dashboard/Calendar/KFCalendar/DayEvent/DayEvent.jsx:46]: ✅ added to modules.css under the circle class

**Review by AnnaSolovykh — APPROVED** [review]

**Review by S-chahine — APPROVED** [review]
Great job Elena!

**Pull request merged**